sql-statements/sql-statement-create-index.md

```diff
@@ -232,6 +232,120 @@ Therefore, when the query performance outweighs the insert and update performanc
 
 Expression indexes have the same syntax and limitations as in MySQL. They are implemented by creating indexes on generated virtual columns that are invisible, so the supported expressions inherit all [limitations of virtual generated columns](/generated-columns.md#limitations).
 
+## Multi-valued index
+
+> **Warning:**
+>
+> For the current version, this feature is still experimental and not recommended for production environments.
+
+Multi-valued index is a kind of secondary index defined on an array column. In an ordinary index, one index record corresponds to one data record (1:1). In a multi-valued index, multiple index records correspond to one data record (N:1). Multi-valued indexes are used to index JSON arrays. For example, a multi-valued index defined on the `zipcode` field will generate one index record for each element in the `zipcode` array.
```

Comment:
is it better to use `normal` instead of `ordinary`? (from MySQL doc, they use `normal index`)

Reply:
done